.gitignore

```diff
@@ -13,4 +13,5 @@ coverage/
 index.html
 .mailmap
 bower.json
-component.json
\ No newline at end of file
+component.json
+debug/local/
```

Comment:
I've a bunch of debug pages that I don't want to push, just added them here to prevent committing them by mistake.